packages/system/src/Name/DetailModal/DetailModal.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { FC, useEffect } from 'react';
+import {
+  useTranslation,
+  useBreadcrumbs,
+  DetailContainer,
+  DetailInputWithLabelRow,
+  DetailSection,
+  Checkbox,
+  Grid,
+  Formatter,
+  Typography,
+  Box,
+  BasicSpinner,
+} from '@openmsupply-client/common';
+import { Link } from '@mui/material';
```

Comment:
Yup, gotcha. I actually tried to use the React Router `Link` first, but I could get it to handle external links properly (it would always add the link to the local host root 🙄)

Anyway, `MuiLink` in "common" now.